lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -368,7 +379,7 @@ def restore_snapshot(self, snapshot: bytes) -> requests.Response:
         Uses force_restore_raft_snapshot to restore the snapshot
         even if the unseal key used at backup time is different from the current one.
         """
-        return self._client.sys.force_restore_raft_snapshot(snapshot)
+        return self._client.sys.restore_raft_snapshot(snapshot)
```

Comment:
This change doesn't seem related to the scope of the PR (manual unseal)

Reply:
this was for debugging. I don't think it matters. reversed it back.

Reply:
But with this change now we won't be able to restore a Vault snapshot on a Vault cluster other than the one the snapshot was taken on in the first place. We won't be able to restore on a new deployment.
Force is used to allow restoring a snapshot of a Vault that use an unseal key X on a new Vault that uses an unseal key Y then unseal it with X once the snapshot is restored.
I'd not remove, but keep it and make sure it works.

Reply:
I see it is not applied, never mind

Reply:
We're definitely gonna support that. I'm resolving this thread since this is outdated.